docs/content_management/field_types/field_type_reference/taxonomyentryfield.md

```diff
@@ -0,0 +1,74 @@
+# TaxonomyEntry Field Type
+
+TaxonomyEntry is a general purpose Field Type that can store only one taxonomy entry (for example, tag or product category). 
+It is used as a parent while creating a tag or category.
+
+| Name           | Internal name         | Expected input |
+|----------------|-----------------------|----------------|
+| `TaxonomyEntry`| `ibexa_taxonomy_entry`| `array`|
+
+## PHP API Field Type 
+
+### Input expectations
+
+Field type `TaxonomyEntry` accepts an array with Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry object.
+
+| Type            | Example         |
+|-----------------|-----------------|
+| `TaxonomyEntry` | See below. |
+
+Example array:
+
+``` php
+new FieldType\TaxonomyEntry\Value(
+	new TaxonomyEntry(
+            ...
+        )
+);
+```
```

Comment:
I looked into Matrix FieldType documentation and I'd change it accordingly:

````suggestion
| Type     | Description | Example         |
|--------|-----------------|-----------------|
| `array` | array with `Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry` object under `taxonomy_entry` key | see below. |

Example using `Ibexa\Taxonomy\FieldType\TaxonomyEntry\Value` object:

``` php
$taxonomyEntry = $this->taxonomyService->loadEntryByIdentifier('example_entry', 'tags');

new \Ibexa\Taxonomy\FieldType\TaxonomyEntry\Value(
	new TaxonomyEntry(
            $taxonomyEntry
        )
);
```

Example using array:
``` php
[
    'taxonomy_entry' => $taxonomyEntry, // load Entry using TaxonomyService
]
```
````